setup.cfg

```diff
@@ -15,3 +16,6 @@ strict_optional = False
 show_none_errors = False
 show_error_codes = True
 no_site_packages = True
+
+[mypy-test.*]
+ignore_errors = True
```

Comment:
Which specific rules do we want to enable here? When enabled, there are over 400 errors - mostly `no-untyped-def`, and `no-untyped-call`

Also, if `test_jupyter` is not excluded, there are a lot errors in `qiskit_ibm_runtime/visualization` and `qiskit_ibm_runtime/jupyter`

Reply:
`qiskit_ibm_runtime/visualization` and `qiskit_ibm_runtime/jupyter` are dead code paths that we will follow up on via https://github.com/Qiskit/qiskit-ibm-runtime/issues/112, https://github.com/Qiskit/qiskit-ibm-runtime/issues/124. So I agree to exclude it from type checking.

I suggest to start with
```
[mypy-test.*]
disallow_untyped_calls = False
disallow_untyped_defs = False
```
and investigate/fix all remaining 60 issues. 41 issues can be fixed very easily by adding an explicit return type (e.g. `-> None`).
```
# appears like disallow_untyped_defs does not apply here. If function parameters are partially typed, mypy expects the return type to be explicitly set
test/integration/test_proxies.py:120: error: Function is missing a return type annotation  [no-untyped-def]
```